docs/operator_parameters.md

```diff
@@ -0,0 +1,175 @@
+Postgres Operator Configuration
+===============================
+
+Postgres operator is configured via a ConfigMap defined by the
+*CONFIG_MAP_NAME* environment variable. Variable names are delimiter-separated
+words.
```

Comment:
> Variable names are delimiter-separated words.

1. Do "variable names" refer to the variables in the operator's configmap ?
2. And the only delimiter we actually use on the configmap is the underscore, isn't it ?